recipes-ccsp/ccsp/rdk-ppp-manager.bb

```diff
@@ -25,6 +25,8 @@ CFLAGS_append = " \
     -I ${STAGING_INCDIR}/sysevent \
     -I${STAGING_INCDIR}/utapi \
     -I${STAGING_INCDIR}/utctx \
+    -Wall \
+    -Werror \
```

Comment:
GIT_TAG = "RC1.1.0b" also needs to be updated to a version that fixes all the warnings.